src/one_dragon/base/web/zip_downloader.py

```
@@ -71,8 +74,10 @@ def unzip(self) -> bool:
         if not os.path.exists(zip_file_path):
             return False
 
-        file_utils.unzip_file(zip_file_path=zip_file_path, unzip_dir_path=self.param.save_file_path)
-        log.info(f"解压完成 {zip_file_path}")
+        # 使用指定的解压路径，如果没有指定则使用save_file_path
```

Ruff (0.14.1)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)